lib/blockchain.go

```diff
@@ -3054,6 +3055,59 @@ func (bc *Blockchain) CreateLikeTxn(
 	return txn, totalInput, changeAmount, fees, nil
 }
 
+func (bc *Blockchain) CreateReactTxn(
+	userPublicKey []byte, postHash BlockHash, isRemove bool, emojiReaction rune,
+	minFeeRateNanosPerKB uint64, mempool *DeSoMempool, additionalOutputs []*DeSoOutput) (
+	_txn *MsgDeSoTxn, _totalInput uint64, _changeAmount uint64, _fees uint64,
+	_err error) {
+
+	//TODO Where would be the best place to place the validation function?
```

Comment:
Validation should at a minimum live in the connect react logic. I recommend defining a regex or a validation function that can be called in the connect logic and then re-using that validation elsewhere.